samples/web/content/apprtc/apprtc.py

```diff
@@ -546,5 +579,7 @@ def get(self, room_id):
     ('/register/(\w+)', RegisterPage),
     # TODO(jiayl): Remove support of /room/ when all clients are updated.
     ('/room/(\w+)', RoomPage),
+    ('/room/(\w+)/status', RoomStatusPage),
```

Comment:
We probably don't, was concerned about side effects of calling /register when I added it, and if we wanted to be able to check the status without actually joining the room, but even if we did that, the room might become full before we try to register, so doing it in a single operation makes more sense.